synthtool/languages/node.py

```diff
@@ -283,36 +283,52 @@ def owlbot_main(template_path: Optional[Path] = None):
     """
     logging.basicConfig(level=logging.DEBUG)
     # Load the default version defined in .repo-metadata.json.
-    default_version = json.load(open(".repo-metadata.json", "rt"))["default_version"]
+    default_version = json.load(open(".repo-metadata.json", "rt")).get(
+        "default_version", None
+    )
+    templates_only = json.load(open(".repo-metadata.json", "rt")).get(
```

Comment:
We could instead inspect .OwlBot.yaml to see if there are any copies, and avoid adding another field to .repo-metadata.json.